crates/graphql/src/schema/objects/profile.rs

```diff
@@ -63,6 +67,21 @@ pub struct TwitterUserProfileResponse {
 
 #[graphql_object(Context = AppContext)]
 impl Profile {
+    fn wallet_address(&self, ctx: &AppContext) -> FieldResult<Option<String>> {
+        let db_conn = ctx.db_pool.get()?;
+        let result: Vec<models::TwitterHandle> = twitter_handle_name_services::table
+            .select(twitter_handle_name_services::all_columns)
+            .limit(1)
+            .filter(twitter_handle_name_services::twitter_handle.eq(&self.handle))
+            .load(&db_conn)
+            .context("Failed to load wallet address")?;
+        if result.len() == 0 {
+            return Ok(None);
+        }
+        let matching_item = result.get(0).unwrap();
+        let wallet_address = &matching_item.wallet_address;
+        Ok(Some(wallet_address.to_string()))
+    }
```

Comment:
I'm confused why wallet_address is stored on the Profile struct but also a computed field.

Reply:
You're right! Removed

Reply:
You're right! Removed